src/v2/BaseLoginRouter.js

```diff
@@ -153,7 +153,25 @@ export default Router.extend({
   },
 
   handleIdxResponseFailure (error = {}) {
-    // 1. loosely check whether is IDX error response
+    // special case: `useInteractionCodeFlow` is true but the Org is not using OIE
+    const featureMsg = 'The requested feature is not enabled in this environment.';
+    if (error?.error === 'access_denied' && error.error_description === featureMsg) {
```

Comment:
instead of `error_description` i would use `i18nkey`

Reply:
Unfortunately the mock JSON is the complete server response. This is the error returned in a V1 environment when a client tries to use the interaction code flow.

We could show ALL "access_denied" errors.